src/bag/design/module.py

```diff
@@ -663,6 +663,7 @@ def design_sources_and_loads(self, params_list: Optional[Sequence[Mapping[str, A
                 'vpulse': 'VPULSE{}',
                 'vpwlf': 'VPWLF{}',
                 'vsin': 'VSIN{}',
+                'ideal_balun': 'BAL{}',
```

Comment:
This dictionary is arranged in alphabetical order, so ideal_balun should go earlier.